lib/tizen_builder.dart

```diff
@@ -80,7 +80,9 @@ class TizenBuilder {
         // Used by AotElfBase to generate an AOT snapshot.
         kTargetPlatform: targetPlatform,
         ...buildInfo.toBuildSystemEnvironment(),
-        kDeviceProfile: tizenBuildInfo.deviceProfile,
+        kDeviceProfile: tizenBuildInfo.deviceProfile == 'tizen'
+            ? 'common'
+            : tizenBuildInfo.deviceProfile,
```

Comment:
This conversion seems redundant.

Note: The defines here are used as identifiers for incremental build.